scintilla/lexers/LexUser.cxx

```diff
@@ -808,8 +808,6 @@ static inline void ReColoringCheck(Sci_PositionU & startPos, int & nestedLevel,
         do
         {
             ch = styler.SafeGetCharAt(--startPos);
-            if (startPos == -1)
-                startPos = 0;

```

Comment:
The deletion of these 2 lines shouldn't be in this PR, should it?

Reply:
It should as it is the fix for a warning:
> Fixing warnings in the latter file was done by removing the check that was always false.

Reply:
Yes, I know.
My point is : there's no impact to remove these 2 lines in this PR, whereas there are more modification above these deleted lines in  PR https://github.com/notepad-plus-plus/notepad-plus-plus/pull/10597. Both modifications in the same file but not in the same PR could make regression due to the separated review.